### PR TITLE
Sue's Updates based on testing

### PR DESCRIPTION
**Pull request by smichalski-cs**
Lots of changes. More to come

### CODE REVIEW

**Comment by smichalski-cs** [comment]
There is only one more section that I need to verify if the Azure instructions are accurate: Remove AZ vCPU limits in Azure: See https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quotas.
If this is not accurate, I will need to do some additional changes.

File: README.md [hunk 1]
```diff
@@ -51,16 +56,22 @@ A Dedicated Cortex Instance is a deployment strategy for CognitiveScale's Cortex
...
 
-## Prerequisites for Launching Dedicated Cortex Instance
+## Azure and Dedicated Cortex Instance architecture
 
 After the prerequisite steps are completed, the client configures their stack in Azure Marketplace, deploys it, and verifies the deployment by logging in to their Kubernetes dashboard.  Your CognitiveScale CS1 and SRE representatives provide assistance and direction throughout the deployment process.
 
-  **NOTE**: Instructions for completing prerequisite processes are detailed below.
+Your Dedicated Cortex Instance is configured as follows:
+
+  - **Subscription**: Create a pay-as-you-go Azure Marketplace subscription (prerequisite)
```
Comment (rdeleon-cs) [inline README.md]: Does not necessarily need to be a `pay-as-you-go` subscription only.... can be an actual paid or enterprise subscription in Azure.

File: README.md [hunk 2]
```diff
@@ -51,16 +56,22 @@ A Dedicated Cortex Instance is a deployment strategy for CognitiveScale's Cortex
...
+   - **Main Resource Group**: Within your subscription you must have a main Resource Group (prerequisite) - NOTE: This Resource Group must be NEW with no existing resources.
```
Comment (rdeleon-cs) [inline README.md]: And must be created by the Marketplace flow.

File: README.md [hunk 3]
```diff
@@ -51,16 +56,22 @@ A Dedicated Cortex Instance is a deployment strategy for CognitiveScale's Cortex
...
+## Azure and Dedicated Cortex Instance architecture
 
 After the prerequisite steps are completed, the client configures their stack in Azure Marketplace, deploys it, and verifies the deployment by logging in to their Kubernetes dashboard.  Your CognitiveScale CS1 and SRE representatives provide assistance and direction throughout the deployment process.
 
-  **NOTE**: Instructions for completing prerequisite processes are detailed below.
+Your Dedicated Cortex Instance is configured as follows:
+
+  - **Subscription**: Create a pay-as-you-go Azure Marketplace subscription (prerequisite)
+   - **Main Resource Group**: Within your subscription you must have a main Resource Group (prerequisite) - NOTE: This Resource Group must be NEW with no existing resources. 
+    - **Main DNS Zone**: Within your main resource group you must configure your main DNS Zone (prerequisite)
```
Comment (rdeleon-cs) [inline README.md]: (i.e. - aks.yourcompany.com)

File: README.md [hunk 4]
```diff
@@ -82,91 +93,237 @@ To complete the Dedicated Cortex Instance deployment you must have the abiltiy t
 
 You will require the following details to complete the instantiation process:
 
-  - Subscription: (created by user during subscription setup)
-  - Location: (centralUS) 
-  - Subdomain: (provided by user - registered internet domain name obtained prior to beginning deployment setup in Azure (e.g. example.com) with a domain name registrar (e.g. AWS, godaddy.com, name.com, etc.)
+  - Subscription: (created by user as part of the prerequisites process)
+  - Resource Group (main): (created as part of the prerequisites process)
+  - DNS Zone (main): (created as part of the prerequisite process)
+  - Location: (centralUS)
```
Comment (rdeleon-cs) [inline README.md]: centralus == default

File: README.md [hunk 5]
```diff
@@ -82,91 +93,237 @@ To complete the Dedicated Cortex Instance deployment you must have the abiltiy t
...
+    - 1) In the table displayed click **Microsoft graph**. A list will open on the right.
+    - 2) At the top of the list click the box labelled **Delegated permissions**.
+    - 3) Click **Directory** in the list to open the selectable options.
+    - 4) Check **Directory.Read.All**. 
+    - 5) At the bottom of the list click **Update Permissions**. The Directory.Read.All permission is added to the table on the left.
+    - 6) In the table click **Microsoft graph** again. 
+    - 7) At the top of the list click the box labelled **Application permissions**.
+    - 8) Click **Directory** in the list to open the selectable options.
+    - 9) Check **Directory.Read.All**. 
+    - 10) At the bottom of the list click **Update Permissions**. The Directory.Read.All permission is added to the table on the left.
```
Comment (rdeleon-cs) [inline README.md]: These steps seem right, I assume you are comfortable with the as they provided a successful path for you?

File: README.md [hunk 6]
```diff
@@ -82,91 +93,237 @@ To complete the Dedicated Cortex Instance deployment you must have the abiltiy t
...
+    - **NOTE**: The **User.Read** permission is added to Microsoft Graph by default.
+      
+  - j.) Under the permissions table click the **Grant admin consent for < account >** button.
+
+    Your API permissions should be as follows:
+
+    | **API/Permission Name** | **Type** | **Description** | **Admin Consent Required** |
+    | --- | --- | --- | --- |
+    | Directory.Read.All | Delegated | Read directory data | Yes. Granted for CognitivesScale |
```
Comment (rdeleon-cs) [inline README.md]: Do not think you need "CognitiveScale" on these lines - maybe "CompanyName" instead?

File: README.md (hunk 5 above)
Comment (smichalski-cs) [reply README.md]: yes these steps were verified

File: README.md [hunk 7]
```diff
@@ -82,91 +92,364 @@ To complete the Dedicated Cortex Instance deployment you must have the abiltiy t
...
+  #
+  # Example of usage:
+  # chmod +x obtainSP.sh
+  # ./obtainSP.sh
+  echo "Obtain a Service Principal for one of your Azure Subscriptions."
+  export ROLE="Contributor"
+  export DEFAULT_ACCOUNT=`az account show -o tsv`
+  export SP_NAME=${1:-'main-resource-group'}
+  DEFAULT_ACCOUNT_ID=$(printf %s "$DEFAULT_ACCOUNT" | cut -f2)
+  if [ ! -z "$DEFAULT_ACCOUNT_ID" ]; then
```
Comment (skottaram-cs) [inline README.md]: This has few ` quotes which will create problems while executing .. can you add a separate file that has this script only

File: README.md [hunk 8]
```diff
@@ -82,91 +92,364 @@ To complete the Dedicated Cortex Instance deployment you must have the abiltiy t
...
+  ```
+  - b. Login to Azure CLI in Powershell.
+
+  ``` 
+  PS /Users/{username}/scripts/dir with spaces> Connect-AzAccount
+  WARNING: To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code FGQGJ4MHZ to authenticate.
+
+  Account                    SubscriptionName TenantId                                   Environment
+  -------                    ---------------- --------                                  -----------
+  username@cognitivescale.com CS_MS_Azure      7637ac23-3d98-4d4b-abfa-825d4fec72e4     AzureCloud
```
Comment (skottaram-cs) [inline README.md]: Don't include our real subscription and tenant ids

File: README.md [hunk 9]
```diff
@@ -51,16 +55,22 @@ A Dedicated Cortex Instance is a deployment strategy for CognitiveScale's Cortex
...
+    - **Main DNS Zone**: Within your main resource group you must configure your main DNS Zone (prerequisite)
+     - **DCI Resource Group**: In the CognitiveScale DCI Create setup in the Azure Marketplace you must create a NEW empty resource group for your DCI deployment. 
+      - **DCI DNS Zone** (subdomain): Within the DCI Resource Group you must create a DNS subdomain that references the Main DNS Zone. Your URLs and access to Cortex tools and UIs are based on this DNS Zone-Base Domain. (e.g. aks.yourcompany.com)
 
 ### Before you begin working in Azure:
 
-- Create DNS subdomain zone at domain registrar like GoDaddy, Name Cheap, or Route 53.
-- Verify that you have the follwing CLI utilities installed: (On Mac use homebrew installer to install the utilities: `brew tap derailed/k9s && brew install kubectl azure-cli jq k9s`)
+- Create DNS domain at domain registrar like Google, GoDaddy, Name Cheap, or Route 53. You will need this to create the main DNS Zone. (BASE DOMAIN)
+- Verify that you have the following CLI utilities installed: (On Mac use homebrew installer to install the utilities: `brew tap derailed/k9s && brew install kubectl azure-cli jq k9s`)
```
Comment (rdeleon-cs) [inline README.md]: k9s is a nice to have, just fyi

Comment (smichalski-cs) [reply README.md]: Cool. on the next round, I will make sure it's labeled as optional.

**Comment by rdeleon-cs** [comment]
> There is only one more section that I need to verify if the Azure instructions are accurate: Remove AZ vCPU limits in Azure: See https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quotas.
> If this is not accurate, I will need to do some additional changes.

This appears to be right. @clee-cs, @jtischler-cs  - any issues with the verbiage for creating a support ticket to up the amount of vCPUs?

**Comment by jtischler-cs** [comment]
> > There is only one more section that I need to verify if the Azure instructions are accurate: Remove AZ vCPU limits in Azure: See https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quotas.
> > If this is not accurate, I will need to do some additional changes.
> 
> This appears to be right. @clee-cs, @jtischler-cs - any issues with the verbiage for creating a support ticket to up the amount of vCPUs?

lgtm

**Pull request merged**